app/models/plugins/fundraiser.rb

```diff
@@ -7,7 +7,20 @@ class Plugins::Fundraiser < ActiveRecord::Base
   DEFAULTS = { title: 'Donate now' }
 
   def liquid_data(supplemental_data={})
-    bands = donation_band.present? ? donation_band.internationalize.to_json : "null"
+    donation_band_name = ''
+    backup_id = donation_band.try(:id)
+
+    if supplemental_data.has_key? :url_params
+      donation_band_name = supplemental_data[:url_params][:donation_band]
+    end
+
+    bands = Donations::BandFinder.find_band(donation_band_name, backup_id)
+
+    if bands
+      bands = bands.internationalize.to_json
```

Comment:
Yeah. This gets called by `LiquidRenderer`, which is so generic you need to turn it into something useable by the JS code at a low level. Not idea, but I'm just trying to make it fit within the structure I've been given.